recipe/meta.yaml

```diff
@@ -42,6 +42,11 @@ requirements:
     - pybind11                               # [build_platform != target_platform]
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [linux64 and cuda_compiler_version != "None"]
+    - cuda-nvrtc  # [linux64 and cuda_compiler_version != "None"]
+    - libcufft-dev  # [linux64 and cuda_compiler_version != "None"]
+    - libcusolver-dev  # [linux64 and cuda_compiler_version != "None"]
+    - libcusparse-dev  # [linux64 and cuda_compiler_version != "None"]
+    - cuda-driver-dev  # [linux64 and cuda_compiler_version != "None"]
```

Comment:
The `cuda-driver-dev` package should be included as a dependency of `{{ compiler('cuda') }}`. I think it can be removed.
```suggestion
```